website/docs/gen-ai/training/bionemo.md

```diff
@@ -0,0 +1,244 @@
+
+```bash
+kubectl get nodes
+```
+:::info
+```bash
+NAME                                          STATUS   ROLES    AGE   VERSION
+ip-100-64-243-22.us-west-2.compute.internal   Ready    <none>   47h   v1.29.0-eks-5e0fdde
+ip-100-64-70-74.us-west-2.compute.internal    Ready    <none>   47h   v1.29.0-eks-5e0fdde
+...
+```
+:::
+
+Next, lets verify all the pods are running.
+
+```bash
+kubectl get pods -A
+```
+Make sure training-operator, nvidia-device-plugin and fsx-csi-controller pods are running and healthy.
+
+</CollapsibleContent>
+
+
+### Run BioNeMo Training jobs
+
+Once you've ensured that all components are functioning properly, you can proceed to submit jobs to your clusters.
+
+#### Step1: Initiate the Uniref50 Data Preparation Task:
+
+The first task, named the `uniref50-job.yaml`, involves downloading and partitioning the data to enhance processing efficiency. This task specifically retrieves the `uniref50 dataset` and organizes it within the FSx for Lustre Filesystem. This structured layout is designed for training, testing, and validation purposes. You can learn more about the uniref dataset [here](https://www.uniprot.org/help/uniref).
+
+To execute this job, navigate to the `examples\training` directory and deploy the `uniref50-job.yaml` manifest using the following commands:
+
+```bash
+cd training
+kubectl apply -f uniref50-job.yaml
+```
+
+:::info
+
+It's important to note that this task requires a significant amount of time, typically ranging from 50 to 60 hours. 
+
+:::
+
+To verify its progress, examine the logs generated by the corresponding pod:
+
+
+```bash
+kubectl logs uniref50-download-xnz42
+[NeMo I 2024-02-26 23:02:20 preprocess:289] Download and preprocess of UniRef50 data does not currently use GPU. Workstation or CPU-only instance recommended.
+[NeMo I 2024-02-26 23:02:20 preprocess:115] Data processing can take an hour or more depending on system resources.
+[NeMo I 2024-02-26 23:02:20 preprocess:117] Downloading file from https://ftp.uniprot.org/pub/databases/uniprot/uniref/uniref50/uniref50.fasta.gz...
+[NeMo I 2024-02-26 23:02:20 preprocess:75] Downloading file to /fsx/raw/uniref50.fasta.gz...
+[NeMo I 2024-02-26 23:08:33 preprocess:89] Extracting file to /fsx/raw/uniref50.fasta...
+[NeMo I 2024-02-26 23:12:46 preprocess:311] UniRef50 data processing complete.
+[NeMo I 2024-02-26 23:12:46 preprocess:313] Indexing UniRef50 dataset.
+[NeMo I 2024-02-26 23:16:21 preprocess:319] Writing processed dataset files to /fsx/processed...
+[NeMo I 2024-02-26 23:16:21 preprocess:255] Creating train split...
+```
+
+After finishing this task, the processed dataset will be saved in the `/fsx/processed` directory. Once this is done, we can move forward and start the `pre-training` job by running the following command:
+
+Following this, we can proceed to execute the pre-training job by running:
+
+```bash
+cd training
+kubectl apply -f esm1nv_pretrain-job.yaml
```

Comment:
Show the output of `kubectl get pods` for this job. Explain how many nodes and pods are used by this job.